lib/queuePopulator/IngestionPopulatorMetrics.js

```diff
@@ -0,0 +1,61 @@
+const { ZenkoMetrics } = require('arsenal').metrics;
+
+const ingestionObjectsQueued = ZenkoMetrics.createCounter({
+    name: 'ingestion_objects_queued_total',
+    help: 'Total number of Kafka messages produced by the ingestion populator',
+    labelNames: ['origin', 'locationName', 'raftID'],
```

Comment:
If you use `locationName` and `raftID` as labels, essentially it allows filtering by either of those labels right? If that's the case, I don't see a value of that. I would rather use `locationName-raftID` to decrease cardinality